src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -198,8 +202,7 @@ public void run(ReaperApplicationConfiguration config, Environment environment)
       context.jmxConnectionFactory.setJmxCredentials(jmxCredentials);
     }
 
-    context.repairManager = RepairManager.create(
-        context,
+    context.repairManager = RepairManager.create(context,
```

Comment:
nit-pick: each argument on a separate line please, including the first. (this happens in a few places)

Reply:
yep ... this is me being lazy.  checkstyle would only allow 150 line methods and when I added the cryptograph it bumped it up to 153.  i "shortened" the method a bit to make checkstyle happy.  I'll refactor something in there into its own method aand then put these other lines back to what they were.